doc/changes/changes_2.4.0.md

```diff
@@ -1,11 +1,13 @@
-# virtual-schema-common-lua 2.4.0, released 2022-10-??
+# virtual-schema-common-lua 2.4.0, released 2023-01-??
  
 Code name: Boolean Properties
  
 ## Summary
 
 Release 2.4.0 of `virtual-schema-common-lua` added the methods `isTrue` and `isFalse` for checking Virtual Schema properties.
 
+We also added support for the `IS [NOT] JSON` predicate.
```

Comment:
```suggestion
Support for the `IS [NOT] JSON` predicate was also added.
```

Reply:
Should the release notes state `We`?

Reply:
I prefer that over passive constructions.